minetest/minetest
# Refactor of moon and sun drawing

This PR is just refactoring. You won't see changes in your game. However, this will ease further feature implementation.

- The goal is to refactor the drawing of sun and moon in the sky

The drawing of sun and moon was in a big method called `render` in `src/client/sky.cpp`. Now, there is new methods `draw_sun` and `draw_moon` that do the job. There are also `draw_sky_body` and `place_sky_body` that create and set the position for the vertices of sun and moon. There are just there to reduce the amount of duplicated code. Feel free to find better names.

## To do

Ready for Review.

- [x] Write doc for thes changes


## How to test

The tests to do: run the game and check sun and moon are well displayed.
Set a texture for sun and moon, and check they're used and well displayed.
